advanced_functionality/multi_model_tensorflow/tf2-MME-regression.ipynb

```diff
@@ -0,0 +1,1534 @@
+{
```

Comment:
<p>Is there a purpose to creating and deleting two individual endpoints when the purpose of this notebook is to demonstrate MME? Can the two individual endpoints be removed for the sake of creating a more focused, fast, and inexpensive notebook to run?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2887/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>